server/py/services/api/initial_data.py

```diff
@@ -100,21 +101,25 @@ def init_data(
     logger.info("Creating initial data")
     config.httpdb.state = mlrun.common.schemas.APIStates.migrations_in_progress
 
-    if is_migration_from_scratch or is_migration_needed:
-        try:
-            _perform_schema_migrations(alembic_util)
-            init_db()
-            db_session = create_session()
+    db_session = create_session()
+    try:
+        if is_migration_from_scratch or is_migration_needed:
             try:
+                _perform_schema_migrations(alembic_util)
+                init_db()
                 _add_initial_data(db_session)
                 _perform_data_migrations(db_session)
-            finally:
-                close_session(db_session)
-        except Exception:
-            state = mlrun.common.schemas.APIStates.migrations_failed
-            logger.warning("Migrations failed, changing API state", state=state)
-            config.httpdb.state = state
-            raise
+            except Exception:
+                state = mlrun.common.schemas.APIStates.migrations_failed
+                logger.warning("Migrations failed, changing API state", state=state)
+                config.httpdb.state = state
+                raise
+
+        # initialize system id
+        _init_system_id(db_session)
```

Comment:
thats not accurate, because we need this to be done after the migrations.. becuase in case we are in a flow of clean db, then the init_system_id will try to access the system_metadata table and it will not be there, causing a crash